ngx-fudis/projects/ngx-fudis/src/lib/directives/tooltip/readme.mdx

```diff
@@ -9,17 +10,17 @@ import * as TooltipStories from "./tooltip.stories.ts";
 Tooltip shows contextual help or information about specific elemenet when user hovers, focuses or clicks on it, depending on the given properties.
 Tooltip directive uses Angular Material Tooltip as its base.
 
-## Examples
-
-<Canvas of={TooltipStories.Example}></Canvas>
-
 ## Usage Guideline
 
 Regular tooltip is triggered on mouse hover and on keyboard focus. Tooltip content is forced for keyboard and screen reader users when element receives focus.
 Consider using `tooltipToggle` property with long and alternative text contents which then will be displayed only when clicking and makes tooltip function as clickable pop-over not distracting keyboard or screenreader users.
 
 Tooltip text is added by `tooltip` property. Tooltip position can be adjusted with optional `tooltipPosition` property.
 
+## Examples
+
+<Canvas of={TooltipStories.Example}></Canvas>
+
 ### Related components
```

Comment:
I would just leave Angular Material Tooltip under "Related Components" and move other components under "Currently Supported Components", but need to check the component list. E.g `Section` is not in the public API yet and there are probably other components where tooltip can be added, like `Button`, `CheckboxGroup` etc.